src/methods/modular_programs/hard/q8/CounterChar.java

```diff
@@ -0,0 +1,34 @@
+package methods.modular_programs.hard.q8;
+
+/*
+Write a program with a method called charCounter that asks the user for a sentence and then counts the number of
```

Comment:
Change to: 
Write a program that asks the user to enter a sentence, then passes that sentence to a method called charCounter. The method counts the number of characters in the given String and returns the total as an integer. The main method then prints out the returned number. (Hint: Space/' ' is not considered a character)